glib/src/convert.rs

```diff
@@ -317,6 +317,65 @@ pub fn locale_to_utf8(opsysstring: &[u8]) -> Result<(crate::GString, usize), Cvt
     }
 }
 
+#[doc(alias = "g_utf8_to_ucs4")]
+#[doc(alias = "g_utf8_to_ucs4_fast")]
+pub fn utf8_to_utf32(str: &str) -> Slice<char> {
```

Comment:
```suggestion
pub fn utf8_to_utf32(str: impl AsRef<str>) -> Slice<char> {
```

for consistency with the others maybe